validation/gpo-params_test.go

```diff
@@ -0,0 +1,109 @@
+		11763071:     true, // goerli-dev-0/base-devnet-0      (no public endpoint)
+		11763072:     true, // sepolia-dev-0/base-devnet-0     (no public endpoint)
+		129831238013: true, // goerli-dev-0/conduit-devnet-0   (no ground truth)
+	}
+
+	checkResourceConfig := func(t *testing.T, chain *ChainConfig) {
+		rpcEndpoint := chain.PublicRPC
+		require.NotEmpty(t, rpcEndpoint, "no public endpoint for chain")
+
+		client, err := ethclient.Dial(rpcEndpoint)
+		require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+		contractAddress := predeploys.GasPriceOracleAddr
+
+		var desiredParams GasPriceOracleParams
+		switch chain.Superchain {
+		case "mainnet":
+			desiredParams = OPMainnetGasPriceOracleParams
+		case "goerli":
+			desiredParams = OPGoerliGasPriceOracleParams
+		case "sepolia":
+			desiredParams = OPSepoliaGasPriceOracleParams
+		case "goerli-dev-0":
+			t.Fatalf("no ground truth for superchain %s", chain.Superchain)
+		case "sepolia-dev-0":
+			t.Fatalf("no ground truth for superchain %s", chain.Superchain)
+		default:
+			t.Fatalf("superchain not recognized: %s", chain.Superchain)
+		}
+
+		actualParams, err := getGasPriceOracleParamsWithRetries(context.Background(), contractAddress, client)
+		require.NoErrorf(t, err, "RPC endpoint %s: %s", rpcEndpoint)
+
+		require.Condition(t, func() bool { return (actualParams.Decimals.Cmp(desiredParams.Decimals) == 0) },
+			"incorrect decimals parameter: got %d, wanted %d", actualParams.Decimals, desiredParams.Decimals)
+		require.Condition(t, func() bool { return (actualParams.Overhead.Cmp(desiredParams.Overhead) == 0) },
+			"incorrect overhead parameter: got %d, wanted %d", actualParams.Overhead, desiredParams.Overhead)
+		require.Condition(t, func() bool { return (actualParams.Scalar.Cmp(desiredParams.Scalar) == 0) },
+			"incorrect scalar parameter: got %d, wanted %d", actualParams.Scalar, desiredParams.Scalar)
+
+		t.Logf("gas price oracle params are acceptable")
+
+	}
+
+	for chainID, chain := range OPChains {
+		if !isExcluded[chainID] {
+			t.Run(chain.Name+fmt.Sprintf(" (%d)", chainID), func(t *testing.T) { checkResourceConfig(t, chain) })
+		}
+	}
+}
+
+// getGasPriceOracleParamsWithRetries get the params stored in the contract at addr.
+func getGasPriceOracleParamsWithRetries(ctx context.Context, addr common.Address, client *ethclient.Client) (GasPriceOracleParams, error) {
+	maxAttempts := 3
+	gasPriceOracle, err := bindings.NewGasPriceOracle(addr, client)
+	if err != nil {
+		return GasPriceOracleParams{}, fmt.Errorf("%s: %w", addr, err)
+	}
+
+	decimals, err := retry.Do(ctx, maxAttempts, retry.Exponential(),
+		func() (*big.Int, error) { return gasPriceOracle.Decimals(&bind.CallOpts{Context: ctx}) })
+	if err != nil {
+		return GasPriceOracleParams{}, fmt.Errorf("%s.Decimals(): %w", addr, err)
+	}
+
+	overhead, err := retry.Do(ctx, maxAttempts, retry.Exponential(),
+		func() (*big.Int, error) { return gasPriceOracle.Overhead(&bind.CallOpts{Context: ctx}) })
```

Comment:
You can reuse `CallOpts`

Reply:
[02df254](https://github.com/ethereum-optimism/superchain-registry/pull/80/commits/02df254deb3e5151c84e010bd56eb75e3aefd0e2)